openlibrary/coverstore/code.py

```diff
@@ -255,25 +255,32 @@ def notfound():
         elif key != 'id':
             value = self.query(category, key, value)
 
-        if not value or (value and safeint(value) in config.blocked_covers):
+        value = safeint(value)
+        if value is None or value in config.blocked_covers:
             return notfound()
 
+        if 9_000_000 > value >= 8_000_000 and size == "L":
```

Comment:
Only downscale the files in the tars

```suggestion
        if 8_820_000 > value >= 8_000_000 and size == "L":
```